extension.js

```diff
@@ -195,7 +195,7 @@ class Formatters {
 }
 
 const formatters = new Formatters();
-formatters.configure();
+formatters.configureWorkspace();
```

Comment:
You've changed the method name at the call site, but not at the definition.

Reply:
Probably easiest to just leave it as `configure` for now.

Reply:
i had a different solution i was pursuing, i thought i reverted it